components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/OIDCClaimUtil.java

```diff
@@ -98,6 +98,36 @@ private static String getLocalRoleName(RoleMapping roleMapping) {
         return roleMapping.getLocalRole().getLocalRoleName();
     }
 
+    @Deprecated
```

Comment:
Please mentioned the introduced method if this is deprecated

Reply:
+1. Updated the Javadoc comment.